recipes/plasflow/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "plasflow" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 352d5d7cd73672130e32f3f61c3f2ad95aa684d7aebf7668ab3f2594996c338b
+
+build:
+  number: 0
+  skip: True # [not py35 or osx]
+
+requirements:
+  host:
+    - python
+    - biopython
+    - numpy
+    - pandas
+    - pip
+    - rpy2
+    - scikit-learn
+    - scipy
+    - bioconductor-biostrings
+    - tensorflow
+  run:
+    - python
+    - biopython
+    - numpy
+    - pandas
+    - rpy2
+    - scikit-learn
+    - scipy
+    - bioconductor-biostrings
+    - tensorflow
+test:
+  imports:
+    - plasflow
+  commands:
+    - PlasFlow.py --help
+
+about:
+  home: https://github.com/smaegol/PlasFlow
+  license: GNU General Public v3 or later (GPLv3+)
+  license_family: GPL3
```

Comment:
license_file: LICENSE

Reply:
this fails with `ValueError: License file given in about/license_file does not exist in source root dir or in recipe root dir (with meta.yaml)` so I've commented it out for now

Reply:
Interesting, maybe because the source root dir is not clear in your case? Does PlasFlow/LICENSE work?